### PR TITLE
Add response_validation_exclude_routes support

### PR DESCRIPTION
**Pull request by raychan**
We want to add the ability to exclude routes for only response validation (while keep request response)

### CODE REVIEW

File: pyramid_swagger/tween.py [hunk 1]
```diff
@@ -462,6 +468,11 @@ def should_exclude_request(settings, request, route_info):
     )
 
 
+def should_exclude_response_validation(settings, route_info):
+    print(route_info)
```
Comment (hippoandy) [inline pyramid_swagger/tween.py:472]: Remove this.

**Review by hippoandy — APPROVED** [review]
LGTM!

File: pyramid_swagger/__about__.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 __summary__ = "Swagger tools for use in pyramid webapps"
 __uri__ = "https://github.com/striglia/pyramid_swagger"
 
-__version__ = "2.8.0"
```
Comment (ymilki) [inline pyramid_swagger/__about__.py:16]: Can you seperate the version change and changelog change into follow-up PR? This way, the changes can be treated independently of the version change.

**Review by ymilki — CHANGES_REQUESTED** [review]

File: pyramid_swagger/__about__.py (hunk 2 above)
Comment (raychan) [reply pyramid_swagger/__about__.py]: done

**Review by hippoandy — APPROVED** [review]

**Review by ymilki — APPROVED** [review]

**Pull request merged**